tests/bin/install-wp-tests.sh

```diff
@@ -88,7 +88,7 @@ install_wp() {
 		else
 			local ARCHIVE_NAME="wordpress-$WP_VERSION"
 		fi
-		download https://wordpress.org/${ARCHIVE_NAME}.tar.gz  $TMPDIR/wordpress.tar.gz
+		download https://wordpress.org/${ARCHIVE_NAME}.tar.gz $TMPDIR/wordpress.tar.gz
```

Comment:
Why was this line changed? This should be a mirror of the "main" script which lives in wp-cli's scaffolding.

Reply:
Oh, I had an error from something unrelated, noticed the double space, and might've just been tidying it. But if it's a mirror I'll restore 😉